lib/module.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+const {
+  ObjectDefineProperty,
```

Comment:
To fix the linter error
```suggestion
  ObjectDefineProperty,
  Set,
```